server/odc-service/src/main/java/com/oceanbase/odc/service/collaboration/environment/EnvironmentService.java

```diff
@@ -84,12 +88,31 @@ public class EnvironmentService {
 
     private final List<Consumer<EnvironmentDeleteEvent>> preDeleteHooks = new ArrayList<>();
     private final List<Consumer<EnvironmentDisableEvent>> preDisableHooks = new ArrayList<>();
+    private final Set<String> DEFAULT_ENV_NAMES = new HashSet<>();
```

Comment:
may use case insensitve tree map,  for handle DEV / SIT / ..

Reply:
okay